### PR TITLE
feature: Support Arduino in Wokwi 

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
Adds support for pytest-embedded-wokwi to run with Arduino build files.

### CODE REVIEW

File: pytest-embedded/pytest_embedded/dut_factory.py [hunk 1]
```diff
@@ -290,6 +290,27 @@ def _fixture_classes_and_options_fn(
...
+                classes[fixture] = WokwiCLI
+                kwargs[fixture].update({
+                    'wokwi_cli_path': wokwi_cli_path,
+                    'wokwi_timeout': wokwi_timeout,
+                    'msg_queue': msg_queue,
+                    'app': app,
+                    'meta': _meta,
+                    'firmware_resolver': resolver,
+                })
+
```
[the remark this answers is not in the document]
Comment (hfudev) [reply pytest-embedded/pytest_embedded/dut_factory.py:313]: I think I gave you a wrong suggestion earlier, adding a `'firmware_resolver': None` here would be enough. The class/kwargs will be injected later.

(don't forget to remove the duplicated codeblocks here)

**Review by hfudev — CHANGES_REQUESTED** [review]

File: pytest-embedded/pytest_embedded/dut_factory.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (P-R-O-C-H-Y) [reply pytest-embedded/pytest_embedded/dut_factory.py]: Done

**Comment by P-R-O-C-H-Y** [comment]
All fixed and retested.

**Review by hfudev — CHANGES_REQUESTED** [review]
LGTM! only left one question

**Comment by hfudev** [comment]
btw, could you rebase on the main branch? the current commits are quite messy

**Comment by P-R-O-C-H-Y** [comment]
Closing in favour of #277

**Pull request closed without merging**